fiftyone/core/utils.py

```diff
@@ -973,19 +973,6 @@ def __init__(self, total=None, progress=None, quiet=None, **kwargs):
         self._progress = progress
         self._callback = callback
 
-    def __call__(self, iterable):
```

Comment:
No longer needed; this is now handled by https://github.com/voxel51/eta/commit/9b4db83db034fd4f29323ca94bbac2963c3b3c16